python/Ganga/new_tests/GPI/Parser/TestSelect.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+
+from ..GangaUnitTest import GangaUnitTest
+
+class TestSelect(GangaUnitTest):
+
+    def test_SelectJob(self):
+        
+        from Ganga.GPI import Job, jobs
+        j=Job()
+
+        job_slice =  jobs.select(status='new')
+
+        assert(len(job_slice) == 1)
```

Comment:
Don't put brackets around the arguments for `assert`. It is a statement, not a function and does not take arguments in the traditional way. This can cause serious problems if you add a second argument as a message like:
```python
assert(len(job_slice) == 1, 'should find 1 job')
```
as it will treat that as a single argument of a 2-tuple which will always evaluate to `True`